### PR TITLE
chore(install-ses): export types

### PR DESCRIPTION
**Pull request by ctjlewis**
Doing:

```ts
import '@agoric/install-ses'
harden({});
```

Incorrectly indicates to TS that `harden` does not exist because there is no type information letting us know statically that `harden` and `lockdown` are now available as globals as a side effect of `import 'ses'` in that package.

This change simply converts `import 'ses'` to `export * from 'ses'` which produces no runtime changes (unless they add exports—which I think it would be fine to re-export) and makes sure types are available.

### CODE REVIEW

File: packages/install-ses/install-ses.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 /* global globalThis LOCKDOWN_OPTIONS process */
-// 'lockdown' appears on the global as a side-effect of importing 'ses'
-import 'ses';
```
Comment (michaelfig) [inline packages/install-ses/install-ses.js:3]: Please keep these lines, as the order of the imports matters (or at least *will* matter when #3891 lands).

I think it's fine to have both an `import` and `export` directive with the same `'ses'` module specifier.

**Review by michaelfig — CHANGES_REQUESTED** [review]
Thanks for this contribution!  I'm always happy to see better typing.

Just one minor nit wrt import ordering then once tests pass I'd be happy to merge.

**Comment by ctjlewis** [comment]
@michaelfig Thanks! My only concern was, will the import and the export cause the side effects of SES to run twice? I know the shim will check if it has already run, but I was mostly thinking about performance.

Will push up changes and if you don't see this as an issue, it should be good to merge.

**Comment by michaelfig** [comment]
> will the import and the export cause the side effects of SES to run twice? I know the shim will check if it has already run, but I was mostly thinking about performance.

Actually, multiple `imports` (or `export from`s) for the same specifier (such as `'ses'`) are defined in ESM to only evaluate the referenced module once and return the same instance for all of those import/exports.  So, no worries about performance, but it is still worth commenting that we have ordering constraints with `ses` and still want to export all its types.

> Will push up changes and if you don't see this as an issue, it should be good to merge.

Sounds great!

**Comment by michaelfig** [comment]
@ctjlewis thank you for this suggestion!

We've since incorporated it into a permanent home in our upstream [endojs/endo](https://github.com/endojs/endo/blob/master/packages/lockdown/pre.js#L8).  `@agoric/install-ses` has been replaced by `@endo/init` which uses `@endo/lockdown`.

Closing this PR, as the intent has been realised.

**Pull request closed without merging**